src/commands/MathCommand.js

```diff
@@ -0,0 +1,63 @@
+const Command = require("./Command");
+const Discord = require("discord.js");
+const { create, all } = require("mathjs");
+const Translation = require("../Translation");
+const math = create(all, {});
+
+class MathCommand extends Command {
+
+    getName() {
+        return "math";
+    }
+
+    getGroup() {
+        return "utilities";
+    }
+
+    getRoles() {
+        return ["member"];
+    }
+
+    init(bot) {
+        this.client = bot.client;
+    }
+
+    init(bot) {}
+
+    call(args, message) {
+        const messageArgs = args.join(" ").toLowerCase();

```

Comment:
You cant make everything lowecase. What if somebody declare two variables x and X. You make them same.